lib/scripts/controllers/options.js

```diff
@@ -97,6 +99,7 @@ export class OptionCtrl {
       this.blacklistedWords = data.userBlacklistedWords.slice(1, -1).split('|');
       this.blacklistedWebsites = data.blacklist.slice(1, -1).split('|');
       this.userDefinedTranslations = JSON.parse(data.userDefinedTranslations);
+      this.userDefinedTranslationList = this.toList(this.userDefinedTranslations);
```

Comment:
It seems that now we will have the same information stored twice, although in different formats (once as "userDefinedTranslations" and once as "userDefinedTranslationList"). Whenever we store the same information twice, we become vulnerable to bugs, because we may forget to keep the storages snchronized, causing the data to become inconsistent.

If you need "userDefinedTranslationList", maybe we should remove "userDefinedTranslations" completely? Or is there a reason to keep it?